deepinv/physics/tomography.py

```diff
@@ -32,6 +33,11 @@ class Tomography(LinearPhysics):
     :param bool circle: If ``True`` both forward and backward projection will be restricted to pixels inside a circle
         inscribed in the square image.
     :param bool parallel_computation: if True, all projections are performed in parallel. Requires more memory but is faster on GPUs.
+    :param bool adjoint_via_backprop: if True, the adjoint will be computed via ``deepinv.physics.adjoint_function``. Otherwise the inverse Radon transform is used.
+        The inverse Radon transform is computationally cheaper (particularly in memory), but has a small adjoint mismatch.
+        The backprop adjoint is the exact adjoint, but might break random seeds since it backpropagates through ``torch.nn.functional.grid_sample``, see the note `here <https://docs.pytorch.org/docs/stable/generated/torch.nn.functional.grid_sample.html>`_,
```

Comment:
```suggestion
        The backprop adjoint is the exact adjoint, but might break random seeds since it backpropagates through :func:`torch.nn.functional.grid_sample`, see the note `here <https://docs.pytorch.org/docs/stable/generated/torch.nn.functional.grid_sample.html>`_.
```